src/N3Lexer.js

```diff
@@ -300,13 +300,28 @@ export default class N3Lexer {
       case ']':
       case '(':
       case ')':
-      case '{':
       case '}':
         if (!this._lineMode) {
           matchLength = 1;
           type = firstChar;
+          break;
+        }
```

Comment:
```suggestion
        }
        break;
```

This will just end up hitting the break on the next case anyways.